src/sgame/sg_cmds.cpp

```diff
@@ -1644,7 +1644,8 @@ static void Cmd_CallVote_f( gentity_t *ent )
 	char   name[ MAX_NAME_LENGTH ] = "";
 	char   caller[ MAX_NAME_LENGTH ] = "";
 	char   reason[ MAX_TOKEN_CHARS ];
-	int    clientNum = -1;
+	int    targetNum = -1;
+	gclient_t *target = nullptr;
```

Comment:
Better to declare the variables later at the same place they are initialized

Reply:
Unfortunately it's used much later in the function inside a if, and used after the if without a check because this is spaghetti code. I'm afraid there is no choice here.